src/Tracy/Bar.php

```diff
@@ -203,10 +204,15 @@ public function dispatchAssets()
 
 		if ($this->useSession && Helpers::isAjax()) {
 			header('X-Tracy-Ajax: 1'); // session must be already locked
+			$seq = mt_rand(1000, 10000000);
```

Comment:
Maybe I'd add timestamp to that sequence to avoid possible collisions. But probability of collision is really really low even with current code, right?

Reply:
That or increment stored in session will guarantee unique

Reply:
Start timestamp will collide easily for requests, uniqid brings issues with html#id selectors.

Reply:
Why not use `random_int()`?

Reply:
I don't think we need crypto-safe number here

Reply:
> to avoid possible collisions

Well, random_int() is exactly for this. Other seeding solutions are not collision free.

Reply:
Ok, implemented, thx